README.md

```diff
@@ -0,0 +1,81 @@
+# Konbata Converter
+
+Konbata converts .doc and .docx files into Canvas .imscc packages.
```

Comment:
It's a little more specialized than this, as stated below.  It only handles files with a specific naming structure.